docs/quick-start-guide.md

```diff
@@ -99,6 +149,16 @@ val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession(
    `SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<carbon_store_path>", "<local metastore path>")`.
  - Data storage location can be specified by `<carbon_store_path>`, like `/carbon/data/store`, `hdfs://localhost:9000/carbon/data/store` or `s3a://carbon/data/store`.
 
+###### Option 2: Using SparkSession with CarbonExtensions
+Start Spark shell by running the following command in the Spark directory:
+
```

Comment:
Can also add this code for better understanding.
SparkSession spark = SparkSession.builder().config(sc.getConf).enableHiveSupport.config("spark.sql.extensions","org.apache.spark.sql.CarbonInternalExtensions").getOrCreate();

Reply:
done